pyvrp/PenaltyManager.py

```diff
@@ -71,14 +78,14 @@ class PenaltyParams:
         have been found amongst the most recent registrations.
     target_feasible
         Target percentage :math:`p_f \\in [0, 1]` of feasible registrations
-        in the last ``num_registrations_between_penalty_updates``
-        registrations.
+        in the last ``solutions_between_updates`` registrations.
     """
 
-    init_capacity_penalty: int = 20
+    init_load_penalty: int = 20
     init_time_warp_penalty: int = 6
+    init_dist_penalty: int = 6
```

Comment:
I wasn't sure about a good default here. Since the time and distance scales are typically very similar, I figured defaulting our `init_dist_penalty` to `init_time_warp_penalty` (=6) would not be too bad.